docs/terminals/payment.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: First Terminal Payment
+sidebar_label: Payment
+description: Process your first terminal payment
+keywords:
+  - terminals
+  - ordering
+  - configuration
+  - payment
+  - refund
+---
+
+# Process a Card Present Payment
+Once you have [configured a terminal device](https://docs.justifi.tech/terminals/configuration) 
+and [confirmed its `CONNECTED` status](https://docs.justifi.tech/terminals/configuration#check-the-terminal-status) 
+you can verify the terminal via test payment or attempt a payment via API integration.
+
+## Verify the Terminal
+To test the device is configured correctly you can process a test payment of $1 via **"Verify Terminal"** 
+on the JustiFi dashboard:\
+Go to https://app.justifi.ai/ and choose **"Terminals"** from the **"Payments"** dropdown. 
+Click into the specific terminal you want to test and click **"Verify Terminal"** on the detail view.\
+Once you confirm the dialog the terminal device will receive the $1 payment 
+which you can complete with a valid credit card.
+
+After the terminal payment has been completed you can view the payment on the JustiFi dashboard:\
```

Comment:
Is it correct that a test payment (via Verify Terminal) will be listed in the Payments list on the customer dashboard and that it can be refunded via the refund button?

Reply:
Yes